docs/custom_installs/local_rtd_vm.rst

```diff
@@ -5,23 +5,29 @@ Assumptions and Prerequisites
 -----------------------------
 
 * Debian VM provisioned with python 2.7.x
-* All python dependencies and setup tools are installed ::
+* All python dependencies and setup tools are installed
```

Comment:
I suppose that we should keep the single `:` on these cases.

(there are more of these ones in the whole diff)

Reply:
I have add the colons.